index.html

```diff
@@ -12,39 +12,39 @@
 <body>
   <!-- Section 1: white portion/racoon -->
   <section>
-    <hr class="line"></hr>
-    <article id="racoonsection">
+    <hr class="line">
+    <article id="whatIsTopCoat" class="articlestyle1">
       <!-- preliminary id incase we want to scroll -->
-      <hr class="racoonstyle">
-      <figure> <img src="racoonsvghere" width="400" height="400" /></a>
+      <hr class="whatIsTopcoatStyle">
```

Comment:
I think we can get away without the `hr`s we should be able to use pseudo elements (`:before`, `:after`, etc.)